srcpkgs/qodem/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'qodem'
+pkgname=qodem
+version=1.0.1
+revision=1
+build_style=gnu-makefile
+make_build_args="-f ./build/Makefile.generic"
+make_build_target="clean qodem"
+makedepends="ncurses-devel gpm-devel"
+short_desc="Qodem is a re-implementation of the Qmodem serial communication package"
+maintainer="Dakota Richline <drichline@protonmail.com>"
+license="CC0-1.0"
+homepage=https://qodem.sourceforge.net/
+distfiles=https://sourceforge.net/projects/qodem/files/qodem/${version}/qodem-${version}.tar.gz
+checksum=dedc73bfa73ced5c6193f1baf1ffe91f7accaad55a749240db326cebb9323359
+
+do_install() {
+	vbin bin/qodem
+	vman docs/qodem.1
```

Comment:
now that we're using the proper build style, we shouldn't override `do_install` and we don't need to install the bin/man manually
```suggestion
post_install() {
```